core-war/src/main/java/org/silverpeas/web/jobstartpage/servlets/JobStartPagePeasRequestRouter.java

```diff
@@ -174,16 +174,13 @@ public String getDestinationNavBar(String function,
 
     if ("Main".equals(function)) {
       jobStartPageSC.init(); // Only ONCE
-      request.setAttribute("SpaceId", request.getParameter("SpaceId"));
+      final String spaceId = request.getParameter("SpaceId");
+      setSpaceOrHomepage(jobStartPageSC, spaceId);
       destination = "/jobStartPagePeas/jsp/jobStartPage.jsp";
     } else if (function.startsWith("GoToSpace")) {
-      if (StringUtil.isDefined(request.getParameter("Espace"))) {
-        jobStartPageSC.setSpaceId(request.getParameter("Espace"));
-      } else {
-        jobStartPageSC.setSpaceId(null);
-      }
-      request.setAttribute("haveToRefreshMainPage", Boolean.TRUE);
-      destination = "/jobStartPagePeas/jsp/jobStartPageNav.jsp";
+      final String spaceId = request.getParameter("Espace");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Espace" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)